### PR TITLE
Port udp and email

### PR DESCRIPTION
**Pull request by raxyte**
Ports:
- `load_udp`
- `save_udp`
- `save_email`

### CODE REVIEW

File: web/docs/tql2/operators/save_udp.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# save_udp
+
+Saves bytes to a UDP socket.
+
+```tql
+udp endpoint:str, [connect=bool, insert_newlines=bool]
```
Comment (dominiklohmann) [inline web/docs/tql2/operators/save_udp.md:6]:
```suggestion
udp endpoint:str, [listen=bool, insert_newlines=bool]
```

This should be `listen` instead (connect being the default, which is exactly the opposite of `load_udp`.

Note, however, that in the code right now neither of the two are implemented.

File: web/docs/tql2/operators/load_udp.md
Comment (dominiklohmann) [inline web/docs/tql2/operators/load_udp.md:1]: The new docs also need to be linked in the operator overview table.

File: web/docs/tql2/operators/save_udp.md (hunk 1 above)
Comment (raxyte) [reply web/docs/tql2/operators/save_udp.md]: I've undocumented this for now.

File: web/docs/tql2/operators/load_tcp.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ the remote endpoint to connect to. Must be of the form
 `[tcp://]<hostname>:<port>`. Use the hostname `0.0.0.0` o accept connections on
 all interfaces.
 
-### `connect = bool (optional)`
-
-Connect to the endpoint instead of listening at it.
+<!-- ### `connect = bool (optional)` -->
+<!---->
+<!-- Connect to the endpoint instead of listening at it. -->
```
Comment (tobim) [inline web/docs/tql2/operators/load_tcp.md:22]: Just remove it.
```suggestion
```

**Review by tobim — APPROVED** [review]

**Pull request merged**